operator/pkg/translate/translate.go

```diff
@@ -541,15 +560,23 @@ func renderFeatureComponentPathTemplate(tmpl string, componentName name.Componen
 
 // renderResourceComponentPathTemplate renders a template of the form <path>{{.ResourceName}}<path>{{.ContainerName}}<path> with
 // the supplied parameters.
-func (t *Translator) renderResourceComponentPathTemplate(tmpl string, componentName name.ComponentName) (string, error) {
+func (t *Translator) renderResourceComponentPathTemplate(tmpl string, componentName name.ComponentName, addonName string) (string, error) {
+	cn := string(componentName)
+	if componentName == name.AddonComponentName {
```

Comment:
this should be commented. it will be hard to remember why this was done.

Reply:
ok done